azkaban-db/src/main/sql/create.project_versions.sql

```diff
@@ -9,6 +9,7 @@ CREATE TABLE project_versions (
   num_chunks           INT,
   resource_id          VARCHAR(512)  DEFAULT NULL,
   startup_dependencies MEDIUMBLOB    DEFAULT NULL,
+  uploader_ip_addr     VARCHAR(50)   DEFAULT NULL,
```

Comment:
The value of the `X-Forwarded-For` header can easily exceed this limit based on:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For
> If a request goes through multiple proxies, the IP addresses of each successive proxy is listed.

Reply:
Thanks for pointing it out. I think this can be easy fix as we are only focusing on originating IP.